docs/releasing/before-publishing-a-release.md

```diff
@@ -12,7 +12,15 @@ Optionally, we can also let the prototype kit know that we are coordinating a re
 
 ## Raise release issues
 
-Those leading on the release should then raise new issues in the team GitHub repositories ([govuk-frontend](https://github.com/alphagov/govuk-frontend), [govuk-design-system](https://github.com/alphagov/govuk-design-system/issues/2024), [govuk-frontend-docs](https://github.com/alphagov/govuk-frontend-docs) to track the following:
+Release issues should be created in the team GitHub repositories: [govuk-frontend](https://github.com/alphagov/govuk-frontend), [govuk-design-system](https://github.com/alphagov/govuk-design-system/issues/2024) and [govuk-frontend-docs](https://github.com/alphagov/govuk-frontend-docs).
```

Comment:
Similar to #3405, this should probably link to the repo consistently:

```suggestion
Release issues should be created in the team GitHub repositories: [govuk-frontend](https://github.com/alphagov/govuk-frontend), [govuk-design-system](https://github.com/alphagov/govuk-design-system/) and [govuk-frontend-docs](https://github.com/alphagov/govuk-frontend-docs).
```

(Or, if we think it's useful to link to the 'example' issues, we should make that clear in the link text and do it consistently for all of the repos)